modules/pel/peltool/README.md

```diff
@@ -0,0 +1,72 @@
+### peltool Usage
+To get more peltool's capabilities, use the help option `peltool.py --help`.
+
+**Note:** For non-BMC environment user must pass the PEL directory to parse PELs
+          e.g. `peltool.py -p <pel_dir> -l`, `peltool.py -p <pel_dir> -n`, `peltool.py -p <pel_dir> -a`
+
+#### List PELs
+- Get list of PELs : `peltool.py -l` or `peltool.py -ls` (By default serviceable PELs are listed)
+- Get list of PELs including serviceable PELs : `peltool.py -ls`
+- Get list of PELs including non-serviceable PELs: `peltool.py -lN`
+- Get list of PELs including hidden PELs: `peltool.py -lH`
+- Get list of PELs including terminating PELs: `peltool.py -lt`
+- Get list of serviceable, non-serviceable, and hidden PELs: `peltool.py -lsNH`
+
+##### List PELs based on severity and only options
+- Get PELs including PELs with severity as Informational: `peltool.py -l -S Informational`
```

Comment:
nit: since these are '-l' options like you had above, probably best to keep the wording the same, so 'Get list of PELs'.  also, could just change them all to 'List PELs'.